__tests__/intergration.test.js

```diff
@@ -68,3 +68,20 @@ describe('/api/articles/:article_id', () => {
 })
 
 
+describe('POST /api/articles/:article_id/comments', () => {
+    it('should return comment with data provided in body ', () => {
+        const comment = { username: 'butter_bridge', body: 'yay it worked' }
+        return request(app).post('/api/articles/1/comments')
+        .send(comment)
+        .expect(201)
+        .then((response) => {
+            console.log(response.body);
+            expect(response.body).toHaveProperty('comment_id')
+            expect(response.body).toHaveProperty('author', comment.username)
+            expect(response.body).toHaveProperty('article_id')
+            expect(response.body).toHaveProperty('body', comment.body)
+            expect(response.body).toHaveProperty('created_at')
+
+        })
```

Comment:
we also want to check the types of values that come back (including that the article_id is the one that we asked for. 
Have a look into:
```
 expect.objectContaining({
                body: expect.any(String),
```